switchboard/sql/SCHEMA.sql

```diff
@@ -188,7 +188,9 @@ create type exit_status as enum (
     -- host reported success
     'host_terminated_with_success',
     -- job timeout
-    'host_terminated_timeout'
+    'host_terminated_timeout',
+    -- job was stopped by user
+    'stopped_by_user'
```

Comment:
Can we call this cancelled? There should be a clear distinction between `host_terminated_with_success` (which is also a stop operation issued within the job's environment, either by a script or by a user), or this force-cancel operation?

Reply:
I thought that it would be the supervisor closing the job on successful termination, not the switchboard?